pokemon-app/src/components/EnemyPokemon.js

```diff
@@ -1,41 +1,70 @@
-import React from "react";
+import React, { useState} from "react";
 import fightPad from "../assets/fightpad.png";
 import "../styles/EnemyPokemon.css";
 
-const EnemyPokemon = ({ enemyPokemon }) =>
-  !enemyPokemon ? null : (
+const EnemyPokemon = ({ enemyPokemon }) => {
+  const [ enemyStatsOnTop, setEnemyStatsOnTop ] = useState(true)
+
+  return(
     <>
-      <div className="enemy-pokemon">
-        <div className="enemy-pokemon__display">
-          <div className="enemy-pokemon__display-name">
-            <p className="paragraph">{enemyPokemon?.name}</p>
+      {!enemyPokemon ? null : (
+        <><div className="enemy-pokemon">
+          <div className="enemy-pokemon__display">
+            <div className="enemy-pokemon__display-name">
+              <p className="paragraph">{enemyPokemon?.name}</p>
+            </div>
+
+            <div className="enemy-pokemon__display-image">
+              <img
+                src={`${enemyPokemon?.sprites?.front_default}`}
+                alt={`Front default of ${enemyPokemon?.name} sprite`}
+              />
+            </div>
           </div>
 
-          <div className="enemy-pokemon__display-image">
-            <img
-              src={`${enemyPokemon?.sprites?.front_default}`}
-              alt={`Front default of ${enemyPokemon?.name} sprite`}
-            />
+          <div 
+            onClick={() => setEnemyStatsOnTop(true)}
+            className={`enemy-pokemon__stats-title ${enemyStatsOnTop ? "on-tip-top" : ""}`}
+          >Stats:</div>
+
+          <div 
+            onClick={() => setEnemyStatsOnTop(false)}
+            className={`enemy-pokemon__moves-title ${enemyStatsOnTop ? "" : "on-tip-top"}`}
+          >Moves:</div>
+
+          <div className={`enemy-pokemon__stats ${enemyStatsOnTop ? "on-top" : ""}`}>
+            <ol className="enemy-pokemon__stats-list">
+              {!enemyPokemon?.stats
+                ? null
+                : enemyPokemon?.stats.map((statData, statIndex) => (
```

Comment:
Maybe write a function to handle this? Might make the code slightly more readable. Everything is looking great. Super excited to see it in action.